FLEDGE.md

```diff
@@ -933,16 +978,32 @@ The output of `generateBid()` contains the following fields:
     browser to select only some of the returned adComponents in order to help
     make the ad k-anonymous. See [Flexible Component Ad Selection Considering k-Anonymity](#341-flexible-component-ad-selection-considering-k-anonymity)
     for more details.
+*   selectedBuyerAndSellerReportingId: (optional) A string from the interest
+    group's ad's `selectableBuyerAndSellerReportingIds` array; if it's not in
+    the array, the bid is ignored. If present, this is reported to `reportWin()`
+    and `reportResult()` along with `buyerAndSellerReportingId` and
+    `buyerReportingId`; the resulting bid may only win the auction if the
+    selected value is jointly k-anonymous along with `buyerAndSellerReportingId`,
+    `buyerReportingId`, the interest group owner, bidding script URL, and render
+    URL. If present, it indicates the reporting of this bid would not operate
+    correctly were the `selectedBuyerAndSellerReportingId` not presented to
+    `reportWin()` and `reportResult()`. See
+    [Reporting IDs](#54-reporting-ids-in-reporting) for more details.
 
 In case returning multiple bids is supported by the implementation in use,
 `generateBid` may also return up to `browserSignals.multiBidLimit` valid bid
 objects of the format above in an array.
 
 Note: Chrome currently imposes an internal limit of 100 for the length of returned bids sequences.
 
-If none of the produced bids pass the k-anonymity checks, `generateBid` will be
+If none of the produced bids pass the k-anonymity checks (the check on the
+render `url` or, when `selectedBuyerAndSellerReportingId` is returned, the
+check on its value along with selected value is jointly k-anonymous along with
```

Comment:
>check on its value along with selected value is jointly...

Are _its_ and _selected value_ referring to the same thing (the returned `selectedBuyerAndSellerReportingId`)?

Reply:
Fixed up the wording and simplified it.